contracts/upgradeable_contracts/multi_amb_erc20_to_erc677/BasicMultiAMBErc20ToErc677.sol

```diff
@@ -58,7 +58,7 @@ contract BasicMultiAMBErc20ToErc677 is
     * @return patch value of the version
     */
     function getBridgeInterfacesVersion() external pure returns (uint64 major, uint64 minor, uint64 patch) {
-        return (1, 3, 0);
+        return (1, 3, 1);
```

Comment:
Since the functionality of the contract is going to be changed and the changes are not a fix, I believe we can update the version to `1.4.0` here.